06-reproj.Rmd

```diff
@@ -525,17 +525,17 @@ This can have implications for file sizes when raster datasets are saved.
 ]:
 
 ```{r rastercrs, echo=FALSE}
-data_frame(
+tibble(
   CRS = c("WGS84", "Equal-area"),
   nrow = c(nrow(con_raster), nrow(con_raster_ea)),
   ncol = c(ncol(con_raster), ncol(con_raster_ea)),
   ncell = c(ncell(con_raster), ncell(con_raster_ea)),
-  resolution = c(mean(res(cat_raster)), mean(res(cat_raster_wgs84), 
+  resolution = c(mean(res(con_raster)), mean(res(con_raster_ea), 
```

Comment:
You happy with this @jannes-m ? Looks right to me.

Reply:
:+1:

Reply:
Jakub has written that, since he gave his ok, it should be fine

Reply:
Thanks guys!